# Phylogenetic Biology - Final Project

## Guidelines - you can delete this section before submission

This is a stub for your final project. Edit/ delete the text in this readme as needed.

There are two ways you can use this document:  
- You can download this file to a folder on your computer, edit this document and add other files (data, code, etc), and then zip up and submit the folder on canvas.
- You can for the [repository](finalproject) containing this document on gitub. Then commit and push your canges to the repository, and submit a link to the repository on canvas.

Github is a great way to work on projects, but also has a steep initial learning curve.


Some guidelines and tips:

- Use the stubs below to write up your final project. Alternatively, if you would like the writeup to be an executable document (with [knitr](http://yihui.name/knitr/), [jupytr](http://jupyter.org/), or other tools), you can create it as a separate file and put a link to it here in the readme.

- For information on formatting text files with markdown, see https://guides.github.com/features/mastering-markdown/ . You can use markdown to include images in this document by linking to files in the repository, eg `![GitHub Logo](/images/logo.png)`.

- The project must be entirely reproducible. In addition to the results, the repository must include all the data (or links to data) and code needed to reproduce the results.

- If you are working with unpublished data that you would prefer not to publicly share at this time, please contact me to discuss options. In most cases, the data can be anonymized in a way that putting them in a public repo does not compromise your other goals.

- Paste references (including urls) into the reference section, and cite them with the general format (Smith at al. 2003).

OK, here we go.

# Title of my project

## Introduction and Goals

The goal of my project is to answer the question, what is the physiological diversification across the Plethodon genus? Woodland salamanders (Genus Plethodon) are terrestrial and found primarily in the eastern and southern United States. Species can vary widely in geographic range sizes, with several species being montane endemics, restricted to one or a few nearby peaks and others occupying large ranges (Marvin, 1996; Highton, 1995). As lungless salamanders, they rely on cutaneous respiration, and are correspondingly highly sensitive to their environments, specifically to temperature and humidity (to minimize water loss and ensure cutaneous respiration. Understanding Plethodon physiology can generate greater insight into the dynamics of diversification.  A phylogenetic perspective is a crucial component for comparison studies as we must consider species’ degrees of relatedness (Glor, 2010). A proposed chapter of my thesis is to collect tail tips from all the individuals that go through the physiological trials and create a high-quality population- and species-level tree to understand patterns of relatedness and introgression. While a broader salamander tree is available, the genetic information used to build those trees is limited and relatively antiquated when compared with more modern approaches (Kozak et al., 2006; Weisrock et al., 2018; Vietes et al., 2010; Highton et al., 2012). While I will not have the sequences nor all of the trait data available for this project, i hope to use similar methods in order to build my own phylogeny.

The methods I will use to do this are aligning sequences through MAFFT, building a tree through IQTREE, and then I will model the traits through brownian motion or use a disparity through time to see how the physiological traits compare (I will be hopefully be using chapters 8 and 11 for this). 

The data I will use to build the tree will be from mitochondrial data publicly available from NCBI and the physiological trait data will either be from data I have collected in the lab or simulated data for the species I have yet to collect it from. 


## Methods

The tools I used were... 

The sequences are from the NADH dehydrogenase subunit 2 (ND2) gene. I have had a difficult time finding and downloading different species since depending on the gene it depends on how many sequences have been done on multiple species. ND2 gene showed a lot of individuals per taxon, since the sequences are from population studies. Due to this, I am aiming prioritizing have more tips on the tree than more individuals per species and am aiming to get at least 30-40 species on the tree by the final project.

I prepared the sequences for alignment in Virtual Studio Code, then I used MAFT to align all of my sequences. Once my sequences were aligned I used IQTREE to infer the phylogeny.



See analysis files at (links to analysis files).

## Results

The tree in Figure 1...

## Discussion

These results indicate...

The biggest difficulty in implementing these analyses was...

If I did these analyses again, I would...

## References
Glor, R. E. (2010). Phylogenetic insights on adaptive radiation. Annual Review of Ecology, Evolution, and Systematics, 41, 251-270.

Highton, R. (1995). Speciation in eastern North American salamanders of the genus Plethodon. Annual Review of Ecology and Systematics, 26(1), 579-600.

Highton, R., Hastings, A. P., Palmer, C., Watts, R., Hass, C. A., Culver, M., & Arnold, S. J. (2012). Concurrent speciation in the eastern woodland salamanders (genus Plethodon): DNA sequences of the complete albumin nuclear and partial mitochondrial 12s genes. Molecular Phylogenetics and Evolution, 6

Kozak, K. H., Weisrock, D. W., & Larson, A. (2006). Rapid lineage accumulation in a non-adaptive radiation: phylogenetic analysis of diversification rates in eastern North American woodland salamanders (Plethodontidae: Plethodon). Proceedings of the Royal Society B: Biological Sciences, 273(1586), 539-546.

Marvin, G. A. (1996). Life history and population characteristics of the salamander Plethodon kentucki with a review of Plethodon life histories. American Midland Naturalist, 385-400.

Vieites, D. R., Román, S. N., Wake, M. H., & Wake, D. B. (2011). A multigenic perspective on phylogenetic relationships in the largest family of salamanders, the Plethodontidae. Molecular Phylogenetics and Evolution, 59(3), 623-635.

Weisrock, D. W., Hime, P. M., Nunziata, S. O., Jones, K. S., Murphy, M. O., Hotaling, S., & Kratovil, J. D. (2018). Surmounting the large-genome “problem” for genomic data generation in salamanders. In Population genomics: wildlife (pp. 115-142). Springer, Cham.

